README.md

```diff
@@ -15,7 +18,48 @@ The repository includes a copy of the [MDI Library](https://github.com/MolSSI/MD
 
 ## Installation
 
-First clone the repository:
+### Compiling MDI-enabled Tinker
+
+Running calculations with this driver will require an [MDI-enabled fork](https://github.com/taylor-a-barnes/Tinker) of Tinker.
+This can be acquired using:
+
+    git clone --branch mdi-ef https://github.com/taylor-a-barnes/Tinker.git
+
+The above clone should be compiled before running calculations with the driver. First, navigate into the cloned repository. It should be compiled from the `mdi-ef` branch:
+
+    cd Tinker
+    git checkout mdi-ef
+
+The build scripts are in the `dev` folder. If it is the first time you are compiling, you should run `full_build.sh`:
```

Comment:
I would suggest changing the language here to clarify that the build scripts are optional (they just automate some of the normal Tinker build process).  I would say something like "For convenience, build scripts are provided in the `dev` folder.  You can either build Tinker in the normal manner, or you can do:"